main.go

```diff
@@ -193,10 +346,14 @@ func run(cfg *config) {
 }
 
 type containerConfig struct {
-	Name    string   `yaml:"name"`
-	Build   string   `yaml:"build"`
-	Image   string   `yaml:"image"`
-	Command []string `yaml:"command"`
+	Name          string            `yaml:"name"`
+	Build         string            `yaml:"build"`
+	Image         string            `yaml:"image"`
+	Command       []string          `yaml:"command"`
+	Env           map[string]string `yaml:"env"`
+	DockerCompose bool              `yaml:"docker-compose"`
```

Comment:
Might be yagni, but I think docker-compose being a string for the filename might be useful?

Reply:
Yeah, and I _almost_ started there, but then thought "YAGNI" and held back. :)

I promise to add it the first time we need it, deal?

Reply:
yagni ftw 👍